### ISSUE TITLE
Error calling Charge.create from backend with the token provided

### ISSUE THREAD

**Issue by PaoloGi** [jonasbark/flutter_stripe_payment#54]
The token provided by the plugin is always something like "pm_1FQtc6Ghe41SsGqRwgiAqtr4"

in the backend calling the Charge API 
```
 Map<String, Object> chargeParams = new HashMap<>();
        chargeParams.put("amount", 999);
        chargeParams.put("currency", "eur");
        chargeParams.put("description", "some description");
        chargeParams.put("source","pm_1FQtc6Ghe41SsGqRwgiAqtr4");
        try {
            charge = Charge.create(chargeParams);
        } catch (StripeException e) {
            e.printStackTrace();
        }
```

 will always throw the error:

com.stripe.exception.InvalidRequestException: You cannot create a charge with a PaymentMethod. Use the Payment Intents API instead.; request-id: req_z0pj936iwJInrw

**Comment by jonasbark** [comment]
Please provide additional information - how are you getting the token with this package?
Please also check the official Stripe documentation.

**Comment by PaoloGi** [comment]
Flutter impl:
- on main.dart
` StripePayment.setSettings(StripeSettings(publishableKey: "pk_test_mytestkey", merchantIdentifier: "merchant.stripe-example", androidProductionEnvironment: false));`

- on payment click
`var token = await StripePayment.addSource(); //here the token is pm_1FQtc6Ghe41SsGqRwgiAqtr4
//api call
PaymentsService.pay( "userId537", token, "522.37", "test userId537");`

the only way i can make the payment with "pm_1FQtc6Ghe41SsGqRwgiAqtr4 " token  is the PaymentIntent flow

so the backend will be: 


`
```
PaymentIntent paymentIntent = null;
        try {
        List<String> paymentMethodTypes = Arrays.asList("card");
        Map<String, Object> params = new HashMap<>();
        params.put("amount", 2000);
        params.put("currency", "eur");
        params.put("payment_method_types",
                paymentMethodTypes
        );
        paymentIntent = PaymentIntent.create(params);
        Map<String, Object> paramsConfirm = new HashMap<String, Object>();
        paramsConfirm.put("payment_method", "pm_1FQtc6Ghe41SsGqRwgiAqtr4 ");

            paymentIntent = paymentIntent.confirm(paramsConfirm);
        } catch (StripeException e) {
            e.printStackTrace();
        }
```
`

**Comment by jonasbark** [comment]
The next version will fully support the payment intent flow. You can already try it if you want:
https://github.com/jonasbark/flutter_stripe_payment/pull/53

**Comment by jonasbark** [comment]
Fixed with 1.0.0
